insights/parsers/ls_var_lib_pcp.py

```diff
@@ -0,0 +1,38 @@
+"""
+LsVarLibPcp - command ``ls -la /var/lib/pcp``
+=============================================
+
+The ``ls -la /var/lib/pcp`` command provides information for the listing of the ``/var/lib/pcp`` directory.
+
+Attributes:
+    file_permissions (list): list of `FilePermissions` objects for every file from the output
```

Comment:
I don't think this parser provides this attribute.

Reply:
@xiangce I have corrected this mistake, please have a look again